static/scripts/theme-switch.js

```diff
@@ -0,0 +1,31 @@
+function changeThemeTo(val) {
+    document.documentElement.setAttribute("data-theme", val);
+    // save prefs in the browser
+    localStorage.setItem("blog-rust-lang-org-theme", val);
+    // close the theme dropdown
+    const themeChoice = document.getElementById("theme-choice");
+    themeChoice.style.display = 'none';
+}
+
+function dropdown () {
+    const themeChoice = document.getElementById("theme-choice");
+    // says it's more "robust" than using getPropertyValue
+    const val = window.getComputedStyle(themeChoice).display
+    if (val === 'none' || val === '') {
+        themeChoice.style.display = 'block';
+    } else {
+        themeChoice.style.display = 'none';
+    }
```

Comment:
```suggestion
    document.getElementById("theme-choice").classList.toggle("display");
```

And then in your CSS you add:

```css
#theme-choice.display {
    display: block;
}
```

Do as much as you can in CSS.